pkgs/by-name/bu/bulloak/package.nix

```diff
@@ -1,24 +1,52 @@
 { lib
 , fetchFromGitHub
 , rustPlatform
+, fetchurl
+, stdenv
 }:
 
+let
+  # svm-rs-builds requires a list of solc versions to build, and would make network calls if not provided.
+  # The ethereum project does not provide static binaries for aarch64, so we use separate sources, the same as in
+  # svm-rs's source code.
+  solc-versions = {
+    x86_64-linux = fetchurl {
+      url = "https://raw.githubusercontent.com/ethereum/solc-bin/60de887187e5670c715931a82fdff6677b31f0cb/linux-amd64/list.json";
+      hash = "sha256-zm1cdqSP4Y9UQcq9OV8sXxnzr3+TWdc7mdg+Do8Y7WY=";
+    };
+    x86_64-darwin = fetchurl {
+      url = "https://raw.githubusercontent.com/ethereum/solc-bin/60de887187e5670c715931a82fdff6677b31f0cb/macosx-amd64/list.json";
+      hash = "sha256-uUdd5gCG7SHQgAW2DQXemTujb8bUJM27J02WjLkQgek=";
+    };
+    aarch64-linux = fetchurl {
+      url = "https://github.com/nikitastupin/solc/blob/923ab4b852fadc00ffe87bb76fff21d0613bd280/linux/aarch64/list.json";
+      hash = "sha256-bqLP2XIEX8xi/6gO1guUdh7WuAf+aAvNP3arw1uhUh8=";
+    };
+    aarch64-darwin = fetchurl {
+      url = "https://github.com/alloy-rs/solc-builds/raw/260964c1fcae2502c0139070bdc5c83eb7036a68/macosx/aarch64/list.json";
+      hash = "sha256-xrtb3deMDAuDIjzN1pxm5NyW5NW5OyoOHTFsYyWJCYY=";
```

Comment:
seems odd to have 3 different repos to fetch 4 json files... is there a more singular source of truth?

Reply:
As you can see in the [source code](https://github.com/alloy-rs/svm-rs/blob/master/crates/svm-rs/src/releases.rs) for `svm-rs`, those are the same links I used here. The derivation matches the default build behavior.

Reply:
Links were fixed and I checked support for x86_64-darwin just now (had to add some apple sdk buildInput)